balancer-js/src/modules/exits/exits.module.integration-mainnet.spec.ts

```diff
@@ -24,17 +34,16 @@ import { SimulationType } from '../simulation/simulation.module';
 
 dotenv.config();
 
-const TEST_BBEUSD = true;
+const TEST_BBEUSD = false;
```

Comment:
Is there any point keeping the associated code if this is going to be set to false?

Reply:
I believe relayerV5 was deployed only after euler pools stopped working, so it's not trivial to make this test work. In any case, we should be ok by testing other pools once we start supporting unwrapping tokens for multiple linear pool types.

Reply:
I'll remove this test for now.